utils/build-script-impl

```diff
@@ -530,6 +530,10 @@ function set_build_options_for_host() {
     SWIFT_HOST_VARIANT_SDK=$(toupper $platform)
     SWIFT_HOST_VARIANT_ARCH=$architecture
 
+    swift_cmake_options+=(
+      -DSWIFT_EARLY_SWIFT_DRIVER_BUILD=$(build_directory "${LOCAL_HOST}" earlyswiftdriver)/${SWIFT_BUILD_TYPE}/bin
```

Comment:
This seems like it should check that we're building the early swift driver.
You can forward the early swift driver flag in `utils/swift_build_support/swift_build_support/build_script_invocation.py` into the build-script-impl invocation.

You might be able to get a decent idea of what's going on from following the `--skip-build` flag or the zlib/libxml2/curl flags.